packages/cli/src/databases/mysqldb/migrations/1648740597343-LowerCaseEmail.ts

```diff
@@ -0,0 +1,17 @@
+import { MigrationInterface, QueryRunner } from 'typeorm';
```

Comment:
To be more specific, we can rename all three migration files to `{n}-LowerCaseUserEmail`